src/khepri.erl

```diff
@@ -2789,6 +2905,7 @@ info(StoreId) ->
 %% @doc Lists the content of specified store on <em>stdout</em>.
 %%
 %% @param StoreId the name of the Khepri store.
+%% @param Options query options.
```

Comment:
I'll push this one to master separately as well since it's unrelated to the change